src/controllers/legacycontrollermapping.h

```diff
@@ -139,12 +139,12 @@ class LegacyControllerMapping {
     }
 
     inline void setWikiLink(const QString& wikilink) {
-        m_wikilink = wikilink;
+        m_manuallink = wikilink;
         setDirty(true);
     }
 
     inline QString wikilink() const {
-        return m_wikilink;
+        return m_manuallink;
     }
 
```

Comment:
the XML element is still called `wiki` though... so this is essentially an API break and confusing at the same time. I think this refactoring needs to a little deeper. Can you also mass reformat all the controller XMLs to have a manual link instead of a wiki link?

Reply:
If we update the controller API this becomes a main PR. 

Here is also a missing bit: 
https://github.com/mixxxdj/mixxx/blob/b52b6d476ff7925b85e79d732a90bbb6dc599cb8/src/controllers/dlgprefcontroller.cpp#L341

Reply:
sure... its only linting anyways. And the lint is a little questionable as well since nobody actually checks it.

Reply:
Done!